wherehows-common/src/test/java/wherehows/common/utils/JobsUtilTest.java

```diff
@@ -116,6 +101,25 @@ public void testGetEnabledJobsByType() throws IOException {
     Files.deleteIfExists(path2);
   }
 
+  @Test
+  public void testMultipleLinesSameProperties() throws IOException, ConfigurationException {
+    String dir = "/tmp/";
+    String propertyStr =
+        "var1=foo\n" + "var1=foo\n" + "var1=foo\n" + "var2=foo\n" + "var2=foo\n" + "var3=foo\n" + "var1=foo\n"
```

Comment:
Let's add a few variations to the values

Reply:
I have added few tests and enhanced this test.

Reply:
What I meant is not to use "foo" for all values